api-tests/queries-access-control/meta.test.js

```diff
@@ -56,9 +56,8 @@ multiAdapterRunners().map(({ runner, adapterName }) =>
       `,
           });
 
-          expect(query.body).not.toHaveProperty('errors');
```

Comment:
🤔  It's possible for a query to have _both_ `.errors` _and_ `.data`. `.data` could appear to be correct, but it's also important that we check there aren't any `.errors`.

Or is this handled elsewhere now?